### ISSUE TITLE
Comparison to Stanford Scene Graph Parser

### ISSUE THREAD

**Issue by berthyf96** [vacancy/SceneGraphParser#7]
Thanks for the great implementation. Does this function the exact same way as the Stanford Scene Graph Parser? If not, how is it different from the Stanford parser?

**Comment by vacancy** [comment]
No. They are not identical. This repo tries to replicate only part of the results of the _rule-based_ scene graph parser by Schuster et al, with a focus on the empirical performance.

It DOES cover all rules defined implemented in the Stanford Parser, with minor heuristic optimizations and extra rules for handling
- compound nouns
- phrasal prepositions such as "in front of"
- phrasal verbs
- detection of scene-level nouns (e.g., airport)

However, it does NOT cover some features of the Stanford Parser, including:
- quantificational modifiers such as a lot of
- pronoun resolution
- plural nouns

**Comment by berthyf96** [comment]
Thanks for the quick and detailed response!
